### ISSUE TITLE
Add some way to not save a given scrobble edit

### ISSUE THREAD

**Issue by Nixinova** [kawaiiDango/pano-scrobbler#499]
**Is your feature request related to a problem, or particular use cases? Please describe.**

I like to delete the album from scrobbles I listen to from YouTube, SoundCloud etc (non-Spotify). So I am often clicking edit in the app and just removing the text of the album and pressing save. However this then saves the edit to history which I don't want - I end up with ping-ponging edits between "I want an album for this one" and "I don't for this one". I want a way to make one-off edits.

**Describe the solution you'd like**
I want some way to not save a certain edit. Ideas that will accomplish what I want are:

1. a "save edit?" checkbox on the edit page, or
2. a button on the edit page to delete the album without saving it to history, etc etc. or
3. the ability to control which metadata different platforms are able to provide, e.g. "Scrobble Albums from: YouTube ❌ Spotify ✔️" etc.

**Additional info**

**Comment by Lamby777** [comment]
Yeah, and this isn't just for "deleting" scrobbles. Sometimes, I find some scrobble from my computer has the incorrect album listed and open Pano on my phone to edit it real quick... But that saves it for future edits, so I have to go remove the entry just to keep it accurate later.
